# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xjdic23/no-random-return-value.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---


The package failed on PowerPC since it returned a random value in main.

  - Rene Rebe <rene@exactcode.de>

--- ./xjdxgen.c.orig	2003-10-01 09:18:58.000000000 +0200
+++ ./xjdxgen.c	2003-10-01 09:47:06.000000000 +0200
@@ -273,6 +273,7 @@
   jindex[0] = diclen+jiver;
   fwrite(jindex,sizeof(long),indptr+1,fp);
   fclose(fp);
+  exit(0);
 }
 /*======function to sort jindex table====================*/
 void jqsort(long lhs, long rhs)
